code/modules/busy_space/organizations.dm

```diff
@@ -197,8 +240,53 @@
 	work = "research giant"
 	headquarters = "Luna, Sol"
 	motto = ""
+	legit = 95 //they own the local airspace
+
+	missions = list( // they get almost every mission type because they're, you know, NT
+		new /datum/lore/mission/prebuilt/medical("NMV"),
+		new /datum/lore/mission/prebuilt/transport("NTV"),
+		new /datum/lore/mission/prebuilt/defense("NDV"),
+		new /datum/lore/mission/prebuilt/freight("NFV"),
+		new /datum/lore/mission/prebuilt/industrial("NIV"),
+		new /datum/lore/mission/prebuilt/scientific("NSV"),
+		new /datum/lore/mission/prebuilt/salvage("NIV"),
+		new /datum/lore/mission/prebuilt/medical_response("NRV"),
+		new /datum/lore/mission/prebuilt/defense_response("NRV"),
+		)
+
+	serviced = list( //again, protagonists, so this is something close to a map by trade distance of human+tajaran space
+		/datum/lore/system/vir = 20,
+		/datum/lore/system/sol = 15,
+		/datum/lore/system/gavel = 15,
+		/datum/lore/system/oasis = 15,
+		/datum/lore/system/saint_columbia = 10,
+		/datum/lore/system/kess_gendar = 10,
+		/datum/lore/system/tau_ceti = 10,
+		/datum/lore/system/alpha_centauri = 10,
+		/datum/lore/system/new_ohio = 10,
+		/datum/lore/system/new_seoul = 10,
+		/datum/lore/system/el = 10, //specific high NT investment
+		/datum/lore/system/nyx = 10,
+		/datum/lore/system/zhu_que = 5,
+		/datum/lore/system/love = 5,
+		/datum/lore/system/kauqxum = 5,
+		/datum/lore/system/sidhe = 5,
+		/datum/lore/system/mahimahi = 5,
+		/datum/lore/system/relan = 5,
+		/datum/lore/system/exalts_light = 5,
+		/datum/lore/system/rarkajar = 5,
+		/datum/lore/system/raphael = 5, //they own one of the raphaelite colonies
+		/datum/lore/system/mesomori = 2,
+		/datum/lore/system/arrathiir = 2,
+		/datum/lore/system/jahans_post = 2,
+		/datum/lore/system/abels_rest = 2,
+		/datum/lore/system/terminus = 2,
+		/datum/lore/system/eutopia = 2,
+		/datum/lore/system/altair = 2,
+		/datum/lore/system/neon_light = 1,
```

Comment:
Throughout, commas on the last entries may be an issue but I can't speak with absolute code authority.

Reply:
it runs just fine for a couple hours of testing at x60 speed though IDK if it's spamming runtimes or not

Reply:
oops